### ISSUE TITLE
[stable/sentry] Update sentry.conf.py to match current version

### ISSUE THREAD

**Issue by c4tz** [helm/charts#15897]
**Describe the bug**
The version of `sentry.conf.py` in the helm chart is outdated and missing some mappings, such as: https://github.com/getsentry/onpremise/blob/af07ad9c55183b4341c4ad49fe1009cad9dd4135/sentry.conf.py#L291

Because of this, it is not possible to use the Slack integration, for example.

**Which chart**:
Sentry

**What happened**:
I tried to configure Sentry's Slack integration by setting the required env vars like so

```
      --set "web.env[0].name=SENTRY_SLACK_CLIENT_ID"
      --set "web.env[0].value=${SLACK_CLIENT_ID}"
      --set "web.env[1].name=SENTRY_SLACK_CLIENT_SECRET"
      --set "web.env[1].value=${SLACK_CLIENT_SECRET}"
      --set "web.env[2].name=SENTRY_SLACK_VERIFICATION_TOKEN"
      --set "web.env[2].value=${SLACK_VERIFICATION_TOKEN}"
```

When trying to add the Slack integration, I got the error `OAuth Error: Please specify 'client_id'` and there is no client id set as a GET param, because the variable is not set.

**What you expected to happen**:
When adding the slack integration, the client ID should be present and sent to Slack within the authorization request.

**How to reproduce it** (as minimally and precisely as possible):
See "What happened"

**Anything else we need to know**:
By the way, `9.1.2` just came out, you might want to update the appVersion ;)

**Comment by madnight** [comment]
Hi, stale bot. This issue still exists.

**Comment by Gmentsik** [comment]
Hi,

I got the same problem, the current docker version of sentry reads the slack env vars without `SENTRY_` prefix. So just pass the slack configs with `SLACK_CLIENT_ID`
, `SLACK_CLIENT_SECRET` , `SLACK_VERIFICATION_TOKEN`

```
root@sentry-3617047831-3g7wx:/# cat /etc/sentry/sentry.conf.py 
# This file is just Python, with a touch of Django which means
# you can inherit and tweak settings to your hearts content.

# For Docker, the following environment variables are supported:
#  SENTRY_POSTGRES_HOST
#  SENTRY_POSTGRES_PORT
#  SENTRY_DB_NAME
#  SENTRY_DB_USER
#  SENTRY_DB_PASSWORD
#  SENTRY_RABBITMQ_HOST
#  SENTRY_RABBITMQ_USERNAME
#  SENTRY_RABBITMQ_PASSWORD
#  SENTRY_RABBITMQ_VHOST
#  SENTRY_REDIS_HOST
#  SENTRY_REDIS_PASSWORD
#  SENTRY_REDIS_PORT
#  SENTRY_REDIS_DB
#  SENTRY_MEMCACHED_HOST
#  SENTRY_MEMCACHED_PORT
#  SENTRY_FILESTORE_DIR
#  SENTRY_SERVER_EMAIL
#  SENTRY_EMAIL_HOST
#  SENTRY_EMAIL_PORT
#  SENTRY_EMAIL_USER
#  SENTRY_EMAIL_PASSWORD
#  SENTRY_EMAIL_USE_TLS
#  SENTRY_EMAIL_LIST_NAMESPACE
#  SENTRY_ENABLE_EMAIL_REPLIES
#  SENTRY_SMTP_HOSTNAME
#  SENTRY_MAILGUN_API_KEY
#  SENTRY_SINGLE_ORGANIZATION
#  SENTRY_SECRET_KEY
#  SLACK_CLIENT_ID
#  SLACK_CLIENT_SECRET
#  SLACK_VERIFICATION_TOKEN
#  GITHUB_APP_ID
#  GITHUB_API_SECRET
#  BITBUCKET_CONSUMER_KEY
#  BITBUCKET_CONSUMER_SECRET

```

**Comment by c4tz** [comment]
Thank you for the hint, @Gmentsik. I tried to set the env variables without the `SENTRY_`-prefix, but still have no `client_id` in my request and thus get the same error as before.

To investigate further, I had a look at `/etc/sentry/sentry.conf.py` inside the `-web` container started by this Helm chart and it is exactly the same as in the [configmap](https://github.com/helm/charts/blob/master/stable/sentry/templates/configmap.yaml), which is obviously missing the following block from [the current config](https://github.com/getsentry/docker-sentry/blob/master/9.1/sentry.conf.py):

```python
#####################
# SLACK INTEGRATION #
#####################
slack = env('SLACK_CLIENT_ID') and env('SLACK_CLIENT_SECRET')
if slack:
    SENTRY_OPTIONS['slack.client-id'] = env('SLACK_CLIENT_ID')
    SENTRY_OPTIONS['slack.client-secret'] = env('SLACK_CLIENT_SECRET')
    SENTRY_OPTIONS['slack.verification-token'] = env('SLACK_VERIFICATION_TOKEN') or ''
```

But as this is not the only difference, I'm not willing to create a PR for this, as I might break other things with it.

**Comment by mbarrien** [comment]
Resolved in #17754 ?

**Comment by c4tz** [comment]
Yes, it seems to work now. Thank you!
